From 3fe6b82207b984cd83ff7123fc4d2844f391a9ab Mon Sep 17 00:00:00 2001
From: Enrico Berti <hello@enricoberti.com>
Date: Mon, 13 Oct 2014 16:36:17 +0200
Subject: [PATCH 0033/1173] HUE-2401 [search] Visually report selected and
 excluded values for ranges too

Added labels and css classes
---
 apps/search/src/search/templates/search.mako |    8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/apps/search/src/search/templates/search.mako b/apps/search/src/search/templates/search.mako
index 219ec4f..8c15f1d 100644
--- a/apps/search/src/search/templates/search.mako
+++ b/apps/search/src/search/templates/search.mako
@@ -1016,17 +1016,23 @@ ${ dashboard.layout_skeleton() }
         &nbsp;
       </div>
       <div class="content">
+        <strong>${_('selected')}</strong>
         <span data-bind="foreach: $data.properties" style="font-weight: normal">
           <!-- ko if: $.grep($parent.filter(), function(f) { return f.value() == $data.from() && ! f.exclude() }).length > 0 -->
+          <span class="label label-info">
             <strong>${_('from')}</strong> <span data-bind="text: $data.from"></span>
             <strong>${_('to')}</strong> <span data-bind="text: $data.to"></span>
+          </span>
           <!-- /ko -->
         </span>
         <br/>
-        <span data-bind="foreach: $data.properties" style="font-weight: normal"  class="excluded">
+        <strong>${_('excluded')}</strong>
+        <span data-bind="foreach: $data.properties" style="font-weight: normal" class="excluded">
           <!-- ko if: $.grep($parent.filter(), function(f) { return f.value() == $data.from() && f.exclude() }).length > 0 -->
+          <span class="label label-important">
             <strong>${_('from')}</strong> <span data-bind="text: $data.from"></span>
             <strong>${_('to')}</strong> <span data-bind="text: $data.to"></span>
+          </span>
           <!-- /ko -->
         </span>
       </div>
-- 
1.7.9.5

